src/spikeinterface/curation/tests/test_sortingview_curation.py

```diff
@@ -130,9 +138,182 @@ def test_json_curation():
     assert len(sorting_curated_json_mua.unit_ids) == 6
     assert len(sorting_curated_json_mua1.unit_ids) == 5
 
+    print("Test for json curation passed!\n")
+
+
+def test_false_positive_curation():
+    """
+    Test curation for false positives.
+    """
+    # https://spikeinterface.readthedocs.io/en/latest/modules_gallery/core/plot_2_sorting_extractor.html
+    sampling_frequency = 30000.0
+    duration = 20.0
+    num_timepoints = int(sampling_frequency * duration)
+    num_units = 20
+    num_spikes = 1000
+    times = np.int_(np.sort(np.random.uniform(0, num_timepoints, num_spikes)))
+    labels = np.random.randint(1, num_units + 1, size=num_spikes)
+
+    sorting = se.NumpySorting.from_times_labels(times, labels, sampling_frequency)
+    print("Sorting: {}".format(sorting.get_unit_ids()))
+
+    # Test curation JSON:
+    test_json = {"labelsByUnit": {"1": ["accept"], "2": ["artifact"], "12": ["artifact"]}, "mergeGroups": [[2, 12]]}
+
+    json_path = "test_data.json"
```

Comment:
Can you place this in `src/spikeinterface/curation/tests/` and remove it from gitignore? Otherwise CI tests will fail for sure!